### ISSUE TITLE
Loss calculation

### ISSUE THREAD

**Issue by sujeendran** [tianrengao/SqueezeWave#13]
Hi!
Can you help me understand how is the loss calculated and what is the expected good loss value from training the model?

**Comment by tianrengao** [comment]
The expected good loss value for a reasonable result is below -6.0. And according to my experience, after the training loss is below -6, the loss decrease extremely slow, but it does decrease. And the more you train, the better the audio is generated.
